lib/katello/tasks/delete_docker_v1_content.rake

```diff
@@ -0,0 +1,13 @@
+namespace :katello do
+  task :delete_docker_v1_content do
+    docker_repositories = Katello::Repository.where(:content_type => 'docker')
+
+    v1_repos = docker_repositories.select do |repo|
+      Katello.pulp_server.extensions.repository.docker_images(repo.pulp_id).map { |image| image['_ns'] }.uniq == ["units_docker_image"]
```

Comment:
is this field only available in `_ns`? That is a pulp internal field that they may be removing in the future, I'm surprised it bubbles through the API.